src/sql/functions/project.sql

```diff
@@ -777,13 +777,10 @@ CREATE OR REPLACE FUNCTION valid_boundary(_boundary geometry)
 
     SELECT EXISTS(
         SELECT 1
-        WHERE ST_IsValid(_boundary)
-            AND NOT (ST_XMax(_boundary) > 180
-                OR ST_XMin(_boundary) < -180
-                OR ST_YMax(_boundary) > 90
-                OR ST_YMin(_boundary) < -90
-                OR ST_XMax(_boundary) <= ST_XMin(_boundary)
-                OR ST_YMax(_boundary) <= ST_YMin(_boundary))
+        WHERE _boundary IS NOT NULL
+            AND ST_Contains(ST_MakeEnvelope(-180, -90, 180, 90, 4326), _boundary)
+            AND ST_XMax(_boundary) > ST_XMin(_boundary)
+            AND ST_YMax(_boundary) > ST_YMin(_boundary)
```

Comment:
```suggestion
            AND ST_YMax(_boundary) > ST_YMin(_boundary)
```
why no more <=?

Reply:
Min and max cant be the same.  ST_IsValid was catching that before so the logic was fine before.

Reply:
Min and max cant be the same.  ST_IsValid was catching that before so the logic was fine before.